.clangd

```diff
@@ -0,0 +1,2 @@
+CompileFlags:
```

Comment:
It was intended, but I will let others decide if it is useful to include. `clangd` is useful as it can do things like apply formatting on save, etc and it needs this configuration file.

Reply:
I think this should go in `.gitignore` and we can add/modify it locally. I have some emacs plugins that use `clangd` and I'm guessing we might not all want the same configuration.